data/json/items/ammo_types.json

```
@@ -767,6 +767,24 @@
     "name": "12.3ln cartridge",
     "default": "123ln"
   },
+  {
+    "type": "ammunition_type",
+    "id": "273x110",
+    "name": "2.73x110 Diyua",
+    "default": "273x110canister"
+  },
+  {
+    "type": "ammunition_type",
+    "id": "273x44",
+    "name": "2.73x44 Diyua",
+    "default": "273x44HE"
+  },
+  {
+    "type": "ammunition_type",
+    "id": "273x44RG",
+    "name": "2.73x44 Diyua Grenade",
```

```suggestion
    "name": "27.3x44 Diyua Grenade",
```
No skinny grenades for you